tools/c7n_gcp/c7n_gcp/entry.py

```diff
@@ -36,8 +36,8 @@
 import c7n_gcp.resources.source
 import c7n_gcp.resources.spanner
 import c7n_gcp.resources.storage
-import c7n_gcp.resources.sql  # noqa: F401
-
+import c7n_gcp.resources.sql
+import c7n_gcp.resources.deploymentmanager  # noqa: F401
```

Comment:
this is a sorted list, please insert the new resource in the correct position

Reply:
done